hexmap.py

```diff
@@ -4,15 +4,61 @@
 import json
 import sys
 
-
-class World:
-    def __init__(self, worldtype, gas_giant, starport, naval, scout, names):
-        self.worldtype = worldtype
-        self.gas_giant = gas_giant
-        self.starport = starport
-        self.naval = naval
-        self.scout = scout
-        self.names = names
+def number_to_hex(number):
+    if number < 10:
+        return number
+    if number == 10:
+        return "A"
+    if number == 11:
+        return "B"
+    if number == 12:
+        return "C"
+    if number == 13:
+        return "D"
+    if number == 14:
+        return "E"
+    if number == 15:
+        return "F"
+    if number == 16:
+        return "G"

```

Comment:
Unless you *really* want a 'G' instead of a '10', you can use [`hex`](https://docs.python.org/3/library/functions.html#hex).  If you *do* want the 'G', then it's just 1 `if` statement on top of `hex` formatting.  Or I guess you could ditch this method and just add a bunch of `:X` to the f-strings.
```suggestion
def number_to_hex(number):
    return f'{number:X}'
```